Marlin/src/core/serial.cpp

```diff
@@ -37,23 +37,23 @@ PGMSTR(SP_A_STR, " A");  PGMSTR(SP_B_STR, " B");  PGMSTR(SP_C_STR, " C");
 PGMSTR(SP_X_STR, " X");  PGMSTR(SP_Y_STR, " Y");  PGMSTR(SP_Z_STR, " Z");  PGMSTR(SP_E_STR, " E");
 PGMSTR(SP_X_LBL, " X:"); PGMSTR(SP_Y_LBL, " Y:"); PGMSTR(SP_Z_LBL, " Z:"); PGMSTR(SP_E_LBL, " E:");
 
+
+// Hook Meatpack if it's enabled on the first leaf
+#ifdef MEATPACK_ENABLED_ON_SERIAL_PORT_0
```

Comment:
Missing config for `MEATPACK_ENABLED_ON_SERIAL_PORT_0` and `MEATPACK_ENABLED_ON_SERIAL_PORT_1`... lets not forget it...

Reply:
Thanks!